### PR TITLE
Added typings

### PR DESCRIPTION
**Pull request by remojansen**
Implements https://github.com/godaddy/terminus/issues/2

### CODE REVIEW

**Comment by gergelyke** [comment]
Thanks a lot @remojansen ! Can you change your examples to follow the indent size of 2 everywhere?

File: typings/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    type HealthCheck = () => Promise<any>;
+
+    interface HealthCheckMap {
+        [key: string]: HealthCheck;
+    }
+
+    interface TerminusOptions {
+        healthChecks?: HealthCheckMap;
+        onHealthCheck?: HealthCheck;
```
Comment (gergelyke) [inline typings/index.d.ts]: this option is gone, you can remove it

File: package.json [hunk 2]
```diff
@@ -3,8 +3,10 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
+  "typings": "./typings/index.d.ts",
```
Comment (gisenberg) [inline package.json]: Shouldn't this be `types` instead of `typings`? See http://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

**Comment by remojansen** [comment]
Hi @gisenberg  @gergelyke I have changed everything as you requested please take a look :+1:

File: package.json (hunk 2 above)
Comment (remojansen) [reply package.json]: I have some libraries that use typings seems like they have changed this but the old version is still supported. Anyway, I have renamed it to "types".

**Pull request merged**